recipe/build.sh

```diff
@@ -28,13 +73,9 @@ build --toolchain_resolution_debug
 build --define=PREFIX=${PREFIX}
 build --define=PROTOBUF_INCLUDE_PATH=${PREFIX}/include
 build --local_cpu_resources=${CPU_COUNT}"
+build --cxxopt=-I${PREFIX}/include
```

Comment:
surprised this isn't handled in the bazel-toolchain feedstock

Reply:
Just wondering if you had any thoughts on this? Maybe we could put it there if I'm right (I might not be)

Reply:
I do think it should probably be handled in the bazel-toolchain. I don't have the best grasp on custom bazel toolchains however. 

Adding this prevents a compilation error:
```
ERROR: /root/.cache/bazel/_bazel_root/8f43f07b9ed355ced5f93e67f5efb400/external/tsl/tsl/profiler/lib/BUILD:279:11: Compiling tsl/profiler/lib/nvtx_utils.cc failed: undeclared inclusion(s) in rule '@tsl//tsl/profiler/lib:nvtx_utils_impl':
this rule is missing dependency declarations for the following files included by 'tsl/profiler/lib/nvtx_utils.cc':
```

Reply:
This looks like it got slipped in unnecessarily. I've removed it.